src/js/constants/ServiceStatusTypes.js

```diff
@@ -1,8 +1,10 @@
 var ServiceStatusTypes = {
-  RUNNING: 0,
-  DEPLOYING: 1,
-  SUSPENDED: 2,
-  NA: 4
+  NA: 0,
+  RUNNING: 1,
+  DEPLOYING: 2,
+  SUSPENDED: 3,
+  DELAYED: 4,
+  WAITING: 5
```

Comment:
I'm not so sure about the order... Isn't "delayed" more important than "waiting" and "running"/"deploying" more important than "suspended"?